templates/users/login.html

```diff
@@ -1,12 +1,30 @@
 {% extends 'base.html' %}
```

Comment:
Login form is not working for invalid login. It doesn't show any kind of error messages.